### ISSUE TITLE
[Table Viewer] - Accessibility Enhancements

### ISSUE THREAD

**Issue by scottdover** [sassoftware/vscode-sas-extension#1684]
There were a few issues mentioned in https://github.com/sassoftware/vscode-sas-extension/pull/1622 that are less about functionality and more about accessibility. These should be prioritized/addressed after initial column sorting work is completed:

---
UI diff on column name
1.There is no tooltip for type icon, truncated column name, sorting indicator and menu icon
2.Type icon truncation
<img width="447" height="200" alt="image" src="https://github.com/user-attachments/assets/f0f001c1-256e-458c-8b26-8e397f1075f3" />

---

Keyboard accessibility currently doesn't work for the grid menu. Lets address that in this issue
